Doc/reference/datamodel.rst

```diff
@@ -3356,10 +3356,15 @@ left undefined.
    is called if ``type(x).__sub__(x, y)`` returns :data:`NotImplemented` or ``type(y)``
    is a subclass of ``type(x)``. [#]_
 
-   .. index:: pair: built-in function; pow
+   Note that :meth:`__rpow__` should be defined to accept an optional third
+   argument if the ternary version of the built-in :func:`pow` function
```

Comment:
pow() sphinx docs uses terms "the three-argument form" and "the two-argument form".  Maybe you should too (here and below in news/versionchanged).